docs/workloads/azure/data/requirements_data_azure.md

```diff
@@ -12,28 +12,73 @@ keywords:
 ## Local development
 
 * Python 3.9+
-* Poetry https://python-poetry.org/docs/
-* (Windows users) A Linux distribution, e.g. WSL2 https://docs.microsoft.com/en-us/windows/wsl/install
+* [Poetry](https://python-poetry.org/docs/)
+* (Windows users) A Linux distribution, e.g. [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install)
+* Java 8/11/17 as in the [Spark documentation](https://spark.apache.org/docs/latest/)
 
 ## Azure
 
-* Azure subscription - for deploying the solution into
-* Azure service principal - needs permissions to deploy and configure all required resources into the target subscription
-* Azure DevOps project - for running CI/CD pipelines and storing project variables
+* Azure subscription – for deploying the solution into
+* Azure service principal (Application) – needs permissions to deploy and configure all required
+resources into the target subscription
+* Azure DevOps project – for running CI/CD pipelines and storing project variables
 
 ### Azure Pipelines variable groups
 
 Our blueprint solution expects the following [variable groups](https://learn.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=yaml)
 to exist in your Azure DevOps project's Pipelines Library:
 
-* Nonprod Environment:
-  * amido-stacks-de-pipeline-nonprod
-  * amido-stacks-infra-credentials-nonprod
-  * stacks-credentials-nonprod-kv
-* Prod Environment:
-  * amido-stacks-de-pipeline-prod
-  * amido-stacks-infra-credentials-prod
-  * stacks-credentials-prod-kv
+* amido-stacks-de-pipeline-_env_
```

Comment:
I think it would be good if we can show _when_ the variables groups need to exist.  i.e. `amido-stacks-de-pipeline-_env_` has lots of values that will only be known after the platform is deployed, so presume you'll have to add them after you do the initial infra deployment. But possibly the other 2 groups need to exist before the first deployment?

Reply:
You are right, but you are demanding a lot :D I will add this info first thing on Monday morning :)